packaging/centreon-broker-cbd-debuginfo.yaml

```
@@ -0,0 +1,43 @@
+name: "centreon-broker-cbd-debuginfo"
+arch: "${ARCH}"
+platform: "linux"
+version_schema: "none"
+version: "${VERSION}"
+release: "${RELEASE}${DIST}"
+section: "default"
+priority: "optional"
+maintainer: "Centreon <contact@centreon.com>"
+description: |
+  Debuginfo package for centreon-broker-cbd.
+  Commit: @COMMIT_HASH@
+vendor: "Centreon"
+homepage: "https://centreon.com"
```

```suggestion
homepage: "https://www.centreon.com"
```
Currently, https://cenreon.com redirects to http://www.centreon.com. but who knows in the future

---

will be replaced later